ambari-server/src/main/python/ambari_server/setupSecurity.py

```diff
@@ -663,7 +706,7 @@ def setup_ldap(options):
     if input is not None and input != "":
       ldap_property_value_map[ldap_prop.prop_name] = input
 
-  bindAnonymously = ldap_property_value_map["authentication.ldap.bindAnonymously"]
+  bindAnonymously = ldap_property_value_map["ambari.ldap.connectivity.anonymous_bind"]
```

Comment:
There should be a constant created for "ambari.ldap.connectivity.anonymous_bind" to help avoid typo errors.